x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/hooks/use_columns/use_columns.ts

```diff
@@ -306,9 +306,20 @@ export const useColumns = ({
     [columns]
   );
 
+  // remove initialWidth property from the last column to extended it to meet the full page width
+  const columnsWithoutInitialWidthForLastVisibleColumn = useMemo(() => {
+    const lastVisibleColumns = visibleColumns[visibleColumns.length - 1];
+    return columns.map((col) => {
+      if (col.id !== lastVisibleColumns) {
+        return col;
+      }
+      return (({ initialWidth, ...object }) => object)(col);
```

Comment:
How about directly returning the reduced object?

```suggestion
      const { initialWidth, ...rest };
      return rest;
```